python/servo/build_commands.py

```diff
@@ -92,24 +92,36 @@ def notify_darwin(title, text):
         raise Exception("Optional Python module 'pyobjc' is not installed.")
 
 
-def notify_build_done(elapsed, success=True):
+def notify_with_command(command):
+    def notify(title, text):
+        import subprocess
+        subprocess.check_call([command, title, text])
+    return notify
+
+
+def notify_build_done(config, elapsed, success=True):
     """Generate desktop notification when build is complete and the
     elapsed build time was longer than 30 seconds."""
     if elapsed > 30:
-        notify("Servo build", "%s in %s" % ("Completed" if success else "FAILED", format_duration(elapsed)))
+        notify(config, "Servo build",
+               "%s in %s" % ("Completed" if success else "FAILED", format_duration(elapsed)))
 
 
-def notify(title, text):
+def notify(config, title, text):
     """Generate a desktop notification using appropriate means on
     supported platforms Linux, Windows, and Mac OS.  On unsupported
     platforms, this function acts as a no-op."""
```

Comment:
Please update this to say that we can also add a custom command to the config for generating a notification.